server.js

```diff
@@ -79,30 +80,28 @@ app.post('/text', function(req, res, next) {
     var maskedRecipient = recipient.substr(0, recipient.length - 5) + '*****';
 
     // Prepare to send message
-    console.log('Sending message ' + maskedMessage + ' to ' + maskedRecipient + ' from ip ' + ip);
+    log.info('Sending message ' + maskedMessage + ' to ' + maskedRecipient + ' from ip ' + ip);
 
-    var client = new twilio.RestClient(account_sid, auth_token);
+    var client = new twilio(account_sid, auth_token);
 
     var options = {
       to: recipient,
       from: from_number,
-      body: message,
-      statusCallback: null
+      body: message
     };
 
-    client.sendMessage(options, function(err, response) {
-      if (err) {
-        console.error(err);
-        res.sendStatus(503);
-      } else {
-        console.log('Message (' + maskedMessage + ') sent to ' + maskedRecipient);
-        res.sendStatus(200);
-      }
+    client.messages.create(options).then(function() {
+      log.info('message_sent', { message: maskedMessage, recipient: maskedRecipient });
```

Comment:
@dfarache do we really need this? if the server is running well, the logs wont provide us with any useful information